### PR TITLE
fix double dictation process bug, add push-to-dictate option

### PR DESCRIPTION
**Pull request by grimkor**
PR for #9

* Fixes double-dictation bug by storing the subprocess on the class and checking against it before running a new one. 
* Adds push-to-dictate with the L5 button held down to start this feature. Has a toggle in the plugin menu as well as updated "How to" text for the user.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (cboiangiu) [inline pnpm-lock.yaml:1]: please remove this file. keeping `lockfileVersion: '6.0'` for now

**Review by cboiangiu — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (cboiangiu) [reply pnpm-lock.yaml:1]: Changes looking good otherwise 👍

**Comment by cboiangiu** [comment]
Ideally, https://github.com/cboiangiu/decky-dictation/issues/11 should be addressed separately as well at some point, in order to provide a good use experience. Added the issue for visibility. Thank you for your contribution!

File: pnpm-lock.yaml
Comment (grimkor) [reply pnpm-lock.yaml:1]: Reverted the lock to main

**Review by cboiangiu — APPROVED** [review]

**Pull request merged**